gtsfm/bundle/bundle_adjustment.py

```diff
@@ -136,6 +145,37 @@ def create_computation_graph(self, sfm_data_graph: Delayed) -> Delayed:
         return dask.delayed(self.run)(sfm_data_graph)
 
 
+def aggregate_ba_stats(ba_data: GtsfmData) -> Dict[str,Any]:
+    """Create a dictionary of summary statistics for a bundle adjustment result.
+
+    Args:
+        ba_data: bundle adjustment result
+
+    Returns:
+        dictionary containing summary statistics of bundle adjustment result
+    """
+    track_lengths_3d = ba_data.get_track_lengths()
+    scene_reproj_errors = ba_data.get_scene_reprojection_errors()
+
+    convert_to_rounded_float = lambda x: float(np.round(x,3))
+
+    stats_dict = {}
+    stats_dict["number_tracks"] = ba_data.number_tracks()
+    stats_dict["3d_track_lengths"] = {
```

Comment:
just `track_lengths`?

Reply:
I do like differentiating between the 2d and 3d tracks though (and we use the term "3d_track_lengths" in the DA output, and there is a 1:1 correspondence between these two

Reply:
ok cool.